Objects/structseq.c

```diff
@@ -409,16 +395,51 @@ PyStructSequence_InitType(PyTypeObject *type, PyStructSequence_Desc *desc)
 PyTypeObject*
 PyStructSequence_NewType(PyStructSequence_Desc *desc)
 {
-    PyTypeObject *result;
+    PyObject* type;
+    PyObject* bases;
+    PyMemberDef* members;
+
+    PyType_Spec* spec = PyMem_NEW(PyType_Spec, 1);
+    spec->name = desc->name;
+    spec->basicsize = sizeof(PyStructSequence) - sizeof(PyObject *);
+    spec->itemsize = sizeof(PyObject *);
+    spec->flags = Py_TPFLAGS_DEFAULT;
```

Comment:
Should we also set Py_TPFLAGS_HAVE_GC?

Reply:
It didn't have it before but yes, this should definitely be collected by the GC.